data_model/src/domain.rs

```diff
@@ -179,6 +182,8 @@ declare_item! {
         accounts: AccountsMap,
         /// [`Asset`](AssetDefinition)s defined of the `Domain`.
         asset_definitions: AssetDefinitionsMap,
+        /// Total amount of [`Asset`].
+        asset_total_quantities: AssetTotalQuantityMap,
```

Comment:
so you're storing total assets by domain, not by blockchain? Is it possible to mint asset for an account in another domain? like mint `asset#domain1` for `account@domain2`. I think it is possible

Reply:
No, we are storing total amount through blockchain. I just thought to store total assets would be convenient to store inside `Domain` because we already store `AssetsDefinition` this way. 

> I think it is possible

Yeap, this is very possible, i cover such scenario in [test case](https://github.com/hyperledger/iroha/blob/67bdfb030cffe3cee00a90e5034a53174cbe01c9/client/tests/integration/queries/asset.rs#L29) (`white_rabbit@looking_glass` belongs to different domain).

Reply:
Excuse my mistake